src/Sarif.UnitTests/TestData/SarifCurrentToVersionOneVisitor/ExpectedOutputs/TwoResultsWithFixes.sarif

```diff
@@ -156,14 +156,14 @@
         }
       ],
       "rules": {
-        "WEB1079.AttributeValueIsNotQuoted": {
+        "WEB1079": {
           "id": "WEB1079",
           "shortDescription": "The attribute value is not quoted.",
           "messageFormats": {
             "default": "The  value of the '{0}' attribute is not quoted. Wrap the attribute value in single or double quotes."
           }
         },
-        "WEB1066.TagNameIsNotLowercase": {
```

Comment:
>TagNameIsNotLowercase [](start = 17, length = 21)

same note, move the readable names to rule.name: NIT

Reply:
See my response to your other comment on this issue.

---
In reply to: [249572746](https://github.com/Microsoft/sarif-sdk/pull/1228#discussion_r249572746) [](ancestors = 249572746)